ui-components/src/notifications/AlertBox.tsx

```diff
@@ -12,6 +12,7 @@ export interface AlertBoxProps {
   children: ReactNode
   inverted?: boolean
   className?: string
+  fullWidth?: boolean
```

Comment:
Will this change all existing usages of AlertBox since fullWidth defaults to falsy?

Maybe we should call this `boundToLayoutWidth` or something so the functionality doesn't change by default